requests-for-comments/2023-10-31-onboarding-emails.md

```diff
@@ -18,86 +18,105 @@ The 'full' onboarding experience is delivered across some 'sub' campaigns, below
 This campaign triggers when a user completes the `user signed up` event for the first time, on either PostHog US or PostHog EU, provided they also have a valid email address.
 
 1. Wait one hour.
-2. Send Welcome Email. 
+2. Send new Welcome Email. 
```

Comment:
<img width="1006" alt="Screenshot 2023-11-14 at 14 24 17" src="https://github.com/PostHog/meta/assets/84011561/558645a6-8f74-4db9-a0f7-633ee0064e7d">

Reply:
no comment after explainer and "a" before "dashboard template"

Reply:
One thought on this - it's still pretty specific to product analytics. About [half of orgs](https://app.posthog.com/insights/bzzo2PTO) DO onboard to product analytics, but the other half don't. 
![image](https://github.com/PostHog/meta/assets/18598166/21e71ab5-11c5-4888-a5e0-a353eefc603c)

So that means that for half of signups this email about the ARRR funnel isn't super relevant.